### DISCUSSION TITLE
VALIDATE Dynamic form

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#849] [PrimeVue]
Hope this I will get an answer.
So, here is a Stackblitz link: [Validate Form](https://stackblitz.com/edit/k6svjz?file=src%2FApp.vue). Please move to this link and help me to validate that form.

**Comment by tugcekucukoglu** [comment]
Duplicate https://github.com/orgs/primefaces/discussions/816

**Reply by Uzeyir9695** [reply]
I know, but answer was unclear. I couldn't make it work.

**Reply by Uzeyir9695** [reply]
Changing selectedCities to 'value" doesn't make it work as expected, I want to display error messages for each field as well.
